nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -2492,4 +2492,32 @@ public void defaultLibraryModelsObjectNonNull() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void checkForNullSupport() {
+    compilationHelper
+        .addSourceLines(
+            "Nullable.java",
```

Comment:
Why do we need this new test for the `@Nullable` annotation?

Reply:
I wanted to show that the same outputs are created for both SpotBugs annotations. But I can remove this test.

Reply:
Add a comment to that effect (e.g. `// This is just to check the behavior is the same between @Nullable and CheckForNull`) then change the file name to `TestNullable.java` (it's not used elsewhere in the tests, but ideally classes should be in a file with the same name as the class and this is used as the file name that `javac` sees when compiling the test case, I believe 🙂 ).

Other than that, it looks good to go!